manifests/charts/base/templates/clusterrolebinding.yaml

```diff
@@ -27,6 +27,6 @@ roleRef:
   name: istiod-{{ .Values.global.istioNamespace }}
 subjects:
   - kind: ServiceAccount
-    name: istio-pilot-service-account
+    name: istiod-service-account
```

Comment:
This will mean that an "istioctl" in-place upgrade will add new resources + keep the old ones around. NBD but we can document that and add how to clean some old resources.

Reply:
istioctl will prune the old ones (in theory, at least): https://github.com/istio/istio/blob/d0c51437b9cb15b231c5746d7a3b1be2b399646b/operator/pkg/helmreconciler/prune.go#L58

Reply:
...which will break revision upgrades probably now that I think about it

Reply:
Hang on, istioctl unconditionally prunes resources in the "install" namespace?

Reply:
Its not unconditional, its based on some labels. the service account has no revs though. The pruning for istiod takes revs into account

Reply:
so 1.5 -> 1.6 canary will also end of pruning 1.5 SAs? I don't have a istioctl based install else would have verified it.

Reply:
going to test this out now

Reply:
its actually fine because in 1.5 the SA was in pilot chart, now its in `base`... so I guess its fine